boto3/docs/resource.py

```diff
@@ -252,10 +304,21 @@ def _add_waiters(self, section):
                 self._service_name
             )
             documenter = WaiterResourceDocumenter(
-                self._resource, service_waiter_model
+                self._resource, service_waiter_model, self._root_docs_path
             )
             documenter.member_map = self.member_map
             documenter.document_resource_waiters(section)
+            self._add_overview_of_member_type(section, 'waiters')
+
+    def _add_resource_note(self, section):
+        section = section.add_new_section('feature-freeze')
+        section.style.start_note()
+        section.write(
+            "Before using anything on this page, please refer to the resources "
+            ":doc:`user guide <../../../../guide/resources>` for the most recent "
```

Comment:
nit; can we do this as an absolute path relative than relative to the resource page?